models/intercom__company_metrics.sql

```diff
@@ -39,17 +39,17 @@ company_metrics as (
     left join company_enhanced
         on company_enhanced.company_id = contact_company_history.company_id
 
-    group by 1
+    group by company_enhanced.company_id
```

Comment:
For style purposes we like to stick to the group by numberings. Would you be able to update this to still be `group by 1`

Reply:
@fivetran-joemarkiewicz Ah this was an artifact of my group by investigation. Fixed now.